src/content-tags-drawer/data/apiHooks.jsx

```diff
@@ -169,26 +170,32 @@ export const useContentTaxonomyTagsUpdater = (contentId) => {
 
```

Comment:
I think the comments above "This sends messages to the parent page..." needs to be updated. Now the iframe is the child page of the MFE.

Reply:
Updated, good catch